full-service/src/json_rpc/v2/api/wallet.rs

```diff
@@ -139,6 +139,18 @@ where
 {
     global_log::info!("Running command {:?}", command);
 
+    if service.resync_in_progress().map_err(format_error)? {
```

Comment:
Its worth mentioning here that even a single account resyncing would block all API access, even if other accounts are fully synced. I think its acceptable at the moment since resyncing only takes place when the db migration runs, but if we ever added some method to force resyncing of a single account it might be odd that it blocks all access until thats done.

Reply:
Agreed. We currently do have a method for forcing a single account to resync that does _not_ cause the API to lock up, which is expected and intended behavior. Happy to revisit this if you think it's necessary